server/pbenchinacan/pod-nodata.yml

```diff
@@ -78,12 +73,12 @@ spec:
   - command:
     - /opt/keycloak/bin/kc.sh
     - start-dev
+    - --http-port=8090
     - --health-enabled=true
     image: ${PB_INTERNAL_CONTAINER_REG}/pbench/pbenchinacan-keycloak:latest
     name: keycloak
     ports:
-      - containerPort: 8080
-        hostPort: 8090
+      - containerPort: 8090
```

Comment:
Since the clients will need to authenticate directly with the Keycloak broker, I think we do need the `-hostPort: 8090` here.

Reply:
I agree